databases/week3/meal-share-db.sql

```diff
@@ -0,0 +1,90 @@
+CREATE DATABASE IF NOT EXISTS `meal-share-db`;
+USE `meal-share-db`;
+DROP DATABASE IF EXISTS `meal-share-db`;
+
+DESCRIBE meal;
+DROP TABLE meal;
+CREATE TABLE meal(
+id int unsigned NOT NULL AUTO_INCREMENT,
+  title VARCHAR (255) NOT NULL,
+  description TEXT NULL DEFAULT NULL,
+  location  VARCHAR (255) NOT NULL,
+  time DATETIME NULL DEFAULT NULL,

```

Comment:
Not super important, but for the sake of consistency, this field should be named `when` as in https://dbdiagram.io/d/5f0460690425da461f045a29 :)